plugins/GSdx/res/tfx.fx

```diff
@@ -575,6 +589,12 @@ float4 ps_color(PS_INPUT input)
 		c.rgb *= c.rgb < 128./255;
 	}
 
+#if PS_CHANNEL_FETCH == 1
+	float4 t = fetch_blue(input.p.xy);
+#else
+	float4 t = sample_c(uv);
```

Comment:
Here as well.

Reply:
`float4 t = fetch_blue(int2(input.p.xy));`

Reply:
Yeah, you've redefined t here. And you don't use t after this, so it currently does nothing. 😉

Reply:
You need to move this to above line 577, so T gets applied to C.

And instead of `float4 t = sample_c(uv);`, use line 575 `float4 t = sample(input.t.xy, input.t.w);` that's already there.